src/AppBundle/ValueObject/SHA1.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace AppBundle\ValueObject;
+
+final class SHA1
+{
+    private $hash;
+
+    public function __construct(string $hash)
```

Comment:
You could make this constructor private (it's not currently used, and for what use case?), that would avoid the exception handling below.